multinet/api/models/workspace.py

```diff
@@ -23,6 +23,14 @@ class WorkspaceRoleChoice(models.IntegerChoices):
     WRITER = 2
     MAINTAINER = 3
 
+    def get_client_name(self):
+        if self.value == self.READER:
+            return 'reader'
+        elif self.value == self.WRITER:
+            return 'writer'
+        elif self.value == self.MAINTAINER:
+            return 'maintainer'
+
```

Comment:
We can actually remove the need for this function altogether by amending the definition of `WorkspaceRoleChoice`

```python
class WorkspaceRoleChoice(models.IntegerChoices):
    READER = 1, 'reader'
    WRITER = 2, 'writer'
    MAINTAINER = 3, 'maintainer'
```

Then, anywhere we have an instance of `WorkspaceRoleChoice`, we can simply use `choice.label` to get this value. So `WorkspaceRoleChoice.READER.label == 'reader`. [Relevant docs](https://docs.djangoproject.com/en/3.2/ref/models/fields/#choices).

Reply:
Nice! This is good to know.